src/lib/gossip_net/gossip_net.ml

```diff
@@ -313,22 +270,9 @@ module Make (Message : Message_intf) : S with type msg := Message.msg = struct
   let try_call_rpc t (peer : Peer.t) dispatch query =
     (* use error collection, close connection strategy of Tcp.with_connection *)
```

Comment:
can you remove this comment? this code no longer resembles with_connection.

Reply:
:+1: